src/program/symlink.py

```diff
@@ -107,7 +107,8 @@ def run(self, item: Union[Movie, Show, Season, Episode]):
 
     def _calculate_next_attempt(self, item: Union[Movie, Show, Season, Episode]) -> datetime:
         base_delay = timedelta(seconds=5)
-        next_attempt_delay = base_delay * (2 ** item.symlinked_times)
+        delay_seconds = min(100, 5 * (2 ** item.symlinked_times))

```

Comment:
I don't like this change, if symlinks times == 1,

min(100, 5 * 1), remove the the changes to the calculation and the 6th attempt will be 5 * 2^6. If five minutes is too long we could modify the calculation to be base (5) * 2^symlinktimes /2

Reply:
def _calculate_next_attempt(self, item: Union[Movie, Show, Season, Episode]) -> datetime:
        base_delay = timedelta(seconds=4)
        next_attempt_delay = base_delay * (2 ** item.symlinked_times)
        next_attempt_time = datetime.now() + next_attempt_delay
        return next_attempt_time

Reply:
would you prefer it this way:      def _calculate_next_attempt(self, item: Union[Movie, Show, Season, Episode]) -> datetime:
        base_delay = timedelta(seconds=4)
        next_attempt_delay = base_delay * (2 ** item.symlinked_times)
        next_attempt_time = datetime.now() + next_attempt_delay
        return next_attempt_time